modules/sync/protocolbuffers/wellknowntypes/cas/e3629b82d6c4ada936ba8bc4d2e9fb75be20a495a6ad0ef5a42195771630db447b326464756bd7fdd5f491561992eba9fbb601b54543e6bcce18146473e864e4

```diff
@@ -0,0 +1,17 @@
+shake256:0daa29e1d15482bc5f25f3fc93d658073240f23e3d2c2a72829ba55ff32aa02ff321a604b4dcb49bbb97c4b6f157db6d2fa455f53d531d651b67755dc305f6be  LICENSE
+shake256:6823073860a234e46100abe6b97bb31c15ec924a155b63bf746c53a1f6d787356e2e1c235511e94b6c9f78a81bfc54926af48d97dd0944b5a046da47e12a27cd  buf.md
+shake256:50583b7f140a92c3c9a43b8cfa612bd990e1ac70922d3ee83b4804da09bc5f1360e0d6518afdbc3d6760e237a7449658adac7931b9e3d39036d4a57972b5f8e7  buf.yaml
+shake256:1c6874c26892049275074aff5b9f87e59b45a2a6d6c25782ac827d5e64f43f35a5b497e912e0d36ac30eeda2d09363bd9b19e95918d740a98bed15037df65622  google/protobuf/any.proto
+shake256:9281e0bcfc690474b84f95a7607fb581e7b4cb166cdf39b9c893b6899cb756b3a33a7c6284fb6445e8507e8f5ad74379e0ef118791e707612759f6ed42c0bf91  google/protobuf/api.proto
+shake256:a5032b15beec68adf9b22f0bc3187ff17dc3ef27b28ba8d59233ef7170f5ec4cd82e715b2029ed88f935318e9ca2c6abb6633d405ca5057a14c12d30be0ebcf5  google/protobuf/compiler/plugin.proto
+shake256:ea50c00050f22c14d192a1eecc2dfbd0b7ab0a0a11d768959de81387b34907fb076ae4370f4d187854ad2699d1e99c83581371e9006ccc353510410d570a2f37  google/protobuf/cpp_features.proto
+shake256:4f1e14d54f6a4d5fd1ff61dd15642e96c300ae8f557f66a77d05bb7c78754c70d6b6b9327da33b7bd65920dfb2bdafdc2b6f3d6b0537357211ed5d88126463f6  google/protobuf/descriptor.proto
+shake256:95d1785a8277c3f6758e3f188e9432d820e93a99de99d1b45452ed2c8c8b1fecbef49efa8f553641d9bba716993420e255ddbc564d8796d1e480877f8c0c1aec  google/protobuf/duration.proto
+shake256:836c327709e30d7fcf6c35c8cff70c863130e3de6962b6337027f38513a672f1dc0496dd84538ee2c53994cc865fa3d68e0c4a87a4fc974601c7a6ebcdc05fe4  google/protobuf/empty.proto
+shake256:d1706fd968269712822f6b51b3ffab303ccdd33c58b4d6c8414083a4fadb448bfd858d77088649165adfe9a08847f007611a3584718e63884e61f75c8bca7a5c  google/protobuf/field_mask.proto
+shake256:6c1c9c00f4b99d03fbee7fb3de69014840df7ef70adcf1c5c22b81f72ce6b5fc9c292f7b229ea86c8196eb8619945d141320dea6c7e9498e7284afd787106a53  google/protobuf/java_features.proto
+shake256:0fe143d11828bc7a117e201c6643612944849f9547e9a8c4ec748ca230dc8d78312123b3accb8f433c19c3cb30b5849fe46021d217da0db7a557648942c87cd3  google/protobuf/source_context.proto
+shake256:a949db8740c3d3ef65e7787a779db1bd1342767cbd053b510273849cf7cf996f65bd08b9c26086f44d310157b1eee784bac7d0b8f68559a7382132875ed1f30d  google/protobuf/struct.proto
+shake256:52440b4bfea02829f855c8fe9ecb6ec2b00f0b34e5ef371c6f14aaddc1d5873eb115f3fb6f96078fbc1059bda4a8a5f41a5808817817c0bddab94f31f3ba022a  google/protobuf/timestamp.proto
+shake256:32bdc6bbb5e798425b15594f917e79b11608cfac63c9a3621848b217e4bee3a62018860ef41da5e16ee82995c5d13de3459f61d1b295fb4c6fe79cb164521d90  google/protobuf/type.proto
+shake256:2e519a7dd32e17277dfc6b44c5ee17372630d77a8b108b84cb4cb0a440cfc722d28697a9447491cc1ac5412d955b5af11844ccfe7277699bc136673963beca02  google/protobuf/wrappers.proto
```

Comment:
@jhump - This looks like the right list of files for the 27.0 protoc binary artifact.